_config.yml

```diff
@@ -2,7 +2,7 @@
 title:               The ICLR Blog Track
 tagline:             ''
 description:         'For short-term, peer-sourced tests of time, generalizations, specializations, reproductions, etc.!'
-url:                 https://iclr-blog-track.github.io
+url:                 https://iclr-blog-track.github.io/
```

Comment:
Please remove this change (anything outside of the specified files should not be modified)